[Bug Report]: Booklore requres DB ROOT password variable

### App Name

Booklore 

### Version (TrueNAS Catalog App Version)

v1.2.9 

### App Issue

When trying to deploy Booklore, the MariaDB container will fail, and therefore, the Booklore container is not deployed. I already tried to pass env for MYSQL_ROOT_PASSWORD and MARIADB_ROOT_PASSWORD with the same results.

### App Logs

```shell
Container started
2025-09-19 15:38:05+02:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:12.0.2+maria~ubu2404 started.
2025-09-19 15:38:05+02:00 [ERROR] [Entrypoint]: Database is uninitialized and password option is not specified
	You need to specify one of MARIADB_ROOT_PASSWORD, MARIADB_ROOT_PASSWORD_HASH, MARIADB_ALLOW_EMPTY_ROOT_PASSWORD and MARIADB_RANDOM_ROOT_PASSWORD
Container stopped
```

--- a/ix-dev/community/booklore/app.yaml
+++ b/ix-dev/community/booklore/app.yaml
@@ -40,4 +40,4 @@ sources:
 - https://github.com/booklore-app/booklore
 title: BookLore
 train: community
-version: 1.0.0
+version: 1.0.1

--- a/ix-dev/community/booklore/questions.yaml
+++ b/ix-dev/community/booklore/questions.yaml
@@ -28,6 +28,22 @@ questions:
     schema:
       type: dict
       attrs:
+        - variable: db_password
+          label: Database Password
+          description: The database password for Bookstack.
+          schema:
+            type: string
+            default: ""
+            required: true
+            private: true
+        - variable: db_root_password
+          label: Root Database Password
+          description: The root database password for Bookstack.
+          schema:
+            type: string
+            default: ""
+            required: true
+            private: true
         - variable: additional_envs
           label: Additional Environment Variables
           schema:

